.buildconfig.yml

```diff
@@ -248,6 +248,10 @@ projects:
     path: components/support/ktx
     description: 'A set of Kotlin extensions.'
     publish: true
+  support-migration:
```

Comment:
nit: I'm not sure if we should add the `fennec` to the name of the component like `support-fennec-migration` as the `support-migration` name is pretty broad term and everything in this component is related to the `fennec-migration`

Reply:
Technically it can be used to migrate other forks like the TOR browser to an AC-based app too. :)

Reply:
Great 👍